pypeit/onespec.py

```diff
@@ -75,7 +76,7 @@ def from_file(cls, ifile):
         #
         return slf
 
-    def __init__(self, wave, flux, PYP_SPEC=None, ivar=None, mask=None, telluric=None,
+    def __init__(self, wave, wave_grid_mid, flux, PYP_SPEC=None, ivar=None, mask=None, telluric=None,
```

Comment:
Should wave_grid_mid be an optional input here?

Reply:
onespec.py used by coadd.py and telluric.py so far. Coadd.py should always set wave_grid_mid by default while telluric.py assigns wave_grid_mid to the appropriate values, which could be None if operated on spec1dfile and not None if operated on coadd1d files.